api/src/main/java/org/openmrs/module/appointments/model/AdhocTeleconsultationResponse.java

```diff
@@ -0,0 +1,38 @@
+package org.openmrs.module.appointments.model;
+
+import org.openmrs.module.appointments.notification.NotificationResult;
+
+import java.util.List;
+
+public class AdhocTeleconsultationResponse {
```

Comment:
Do we need this class at all? Why can't we use the same appointment class?

Reply:
@angshu Yes, we can use it but added a separate object so adhoc-teleconsultation can scale separately and be independent of appointment.

Reply:
@angshu: Shall we mark this conversation as resolved?